workloads/kube-burner/env.sh

```diff
@@ -17,6 +17,7 @@ export BURST=${BURST:-20}
 export MAX_WAIT_TIMEOUT=${MAX_WAIT_TIMEOUT:-1h}
 export CLEANUP=${CLEANUP:-true}
 export POD_NODE_SELECTOR=${POD_NODE_SELECTOR:-'{node-role.kubernetes.io/worker: }'}
+export VM_NODE_SELECTOR=${VM_NODE_SELECTOR:-'{kubernetes.io/hostname: }'}
```

Comment:
Are you using hostname because you want it to fill certain nodes first? (I see POD_NODE_SELECTOR goes to the general worker role)

Reply:
exactly.